src/main/java/org/folio/marccat/resources/domain/FixedField.java

```diff
@@ -3577,14 +3786,8 @@ public void setSequenceNumber(int sequenceNumber) {
   @Generated("org.jsonschema2pojo")
   public static enum MaterialType {
 
-    MUSIC("music"),
-    MAP("map"),
-    COMPUTER_FILE("computerFile"),
-    BOOK("book"),
-    MIXED_MATERIAL("mixedMaterial"),
-    CONTINUING_RESOURCE("continuingResource"),
-    VISUAL_MATERIAL("visualMaterial"),
-    UNDEFINED("undefined");
+    MUSIC("music"), MAP("map"), COMPUTER_FILE("computerFile"), BOOK("book"), MIXED_MATERIAL("mixedMaterial"),
```

Comment:
Put back as it was before example: 
    MUSIC("music"),
    MAP("map"),
    COMPUTER_FILE("computerFile"),

Reply:
I have changed it

Reply:
I have changed it